lib/utils/logger.js

```diff
@@ -92,13 +98,14 @@ Object.freeze(methods);
 
 /**
  * Prefix log with device id
- * @param {object} deviceDetails
  */
-function devicePrefix(deviceDetails) {
-	if (!deviceDetails || !deviceDetails.shortDisplayName)
+function devicePrefix() {
```

Comment:
I dont like  switching from pure function to dirty. This function now will be hard to test and debugg, because now it completely depends on environment presetup.

Reply:
I humbly disagree. 
This function does not change anything but only computes some result. In this sense I would argue that it is still pure.

Since the intent of this function is to get readable form of the currently used device, so the dependency on the environment is actually specification which is the only one that must be tested. Putting the context as default value would allow for a more ubiquitous function per se, but it would put the mentioned spec into shadow making it easy to overlook for a casual reader.

But this is all just philosophy :) @andriichumak @givehug would you also like me to change this?

Reply:
I'm ok with it, i think it does not do any argument depending computations, if it makes sense :)